release-notes/1.18.2.md

```diff
@@ -5,20 +5,21 @@ The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.18.2.
 Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
 
 ## New features & improvements
-* Android security updates for < insert-month-year > (see Android Security Bulletin - < insert-month-year along with the link to bulletin > for more information).
-* The Android WebView has been updated to < insert-version-along-with-link >.
-
-## Deprecations
-
-< List deprecated features. Provide an alternative, if any, for the users to use instead of the deprecated features >
-
-## Known issues
-
-< List known issues with the current release. Include plans to fix it, if known. >
+* Android security updates for July 2023. See [Android Security Bulletin - July 2023](https://source.android.com/docs/security/bulletin/2023-07-01) for more information.
+* The Android WebView has been updated to [tba](tba).
 
 ## Bug fixes
+* When upgrading Anbox Cloud, the `Reboot required to activate new kernel or GPU drivers` message when running `juju status` does not clear even after rebooting and clearing the notification. <!--AC-1622-->
+* `Honkai: Star Rail` game had alpha blending issues and displayed a pink rectangle during game play. <!--AC-1628-->
+* `Invalid argument` error occurs when configuring kernel parameters during the LXD charm upgrade. <!--AC-1645-->
+* Cannot set a node as unschedulable using the `unschedulable` parameter when updating a node via PATH method to endpoint(1.0/nodes). <!--AC-1689-->
+* Applications requiring camera and microphone do not work in Anbox Cloud version 1.18.0 or later because of a change to the wire protocol between the WebRTC client and server. <!--AC-1710-->
+* In the [Anbox Stream Gateway Charm](https://charmhub.io/anbox-stream-gateway), TLS pinning is not enabled even after the `use_tls_pinning` configuration is set to `true`. <!--AC-1683-->
+* [LP 2024144](https://bugs.launchpad.net/anbox-cloud/+bug/2024144) The Grafana dashboard displays incorrect metrics for streaming stack. <!--AC-1685-->
+* [LP 2020801](https://bugs.launchpad.net/anbox-cloud/+bug/2020801) Accessibility service crashes on Android 13 for Anbox Cloud. <!--AC-1657-->
+* [LP 2024426](https://bugs.launchpad.net/anbox-cloud/+bug/2024426) Anbox Application Registry (AAR) loses metadata on upgrade. <!--AC-1691-->
+* [LP 2024559](https://bugs.launchpad.net/anbox-cloud/+bug/2024559) The LXD storage pool size is not evaluated during the initialisation of Anbox Cloud Appliance. <!--AC-1698-->
 
```

Comment:
As discussed last Friday, we need to call out the bug https://warthogs.atlassian.net/browse/AC-1733 in the 1.18.2 release
and regarding the [LP link](https://bugs.launchpad.net/anbox-cloud/+bug/2025058)

Reply:
Done